frappe/utils/data.py

```diff
@@ -640,7 +640,10 @@ def fmt_money(amount, precision=None, currency=None):
 
 	if currency and frappe.defaults.get_global_default("hide_currency_symbol") != "Yes":
 		symbol = frappe.db.get_value("Currency", currency, "symbol", cache=True) or currency
-		amount = symbol + " " + amount
+		if frappe.db.get_single_value("System Settings", "currency_indicator_style") != "10 USD":
```

Comment:
"10 USD" doesn't look good as a value in code. I'd suggest it should be "Value Currency".